src/server/models.js

```diff
@@ -134,6 +136,37 @@ const createComment = async (user_id, text, date, group_id) => {
   return insertedId;
 }
 
+const getRestaurantsByZipCode = async (zip_code) => {
+  const restaurantsData = restaurants.restaurants;
+
+  const allRestaurants = restaurantsData.reduce((acc, restaurant) => {
+    const formattedRestaurant = {}
+    formattedRestaurant.name = restaurant.restaurant_name;
+    formattedRestaurant.street = restaurant.address.formatted;
+    formattedRestaurant.cuisines = [restaurant.cuisines];
+    formattedRestaurant.hours = restaurant.hours;
+    acc.push(formattedRestaurant)
+    return acc;
+
+  }, [])
+  return allRestaurants;
+}
```

Comment:
@erikeh Lols gonna need the restaurant id

Reply:
restaurant id is included now!